orchestrator-swarm/src/main/java/com/sequenceiq/cloudbreak/orchestrator/swarm/containers/AmbariAgentBootstrap.java

```diff
@@ -63,8 +63,7 @@ public Boolean call() throws Exception {
             List<Bind> binds = new ArrayList<>();
             binds.add(new Bind("/usr/local/public_host_script.sh", new Volume("/etc/ambari-agent/conf/public-hostname.sh")));
             binds.add(new Bind("/data/jars", new Volume("/data/jars")));
-            binds.add(new Bind("/var/log/containers/ambari-agent", new Volume("/var/log/ambari-agent")));
-            binds.add(new Bind("/var/log/containers/consul-watch", new Volume("/var/log/consul-watch")));
+            binds.add(new Bind(String.format("%s/logs/", new ArrayList<>(dataVolumes).get(0)), new Volume("/var/log/")));
```

Comment:
This do not provide that the logs will be stored in the '/hadoopfs/fs1/' folder.